src/main/java/cultfit/FitnessProgram.java

```diff
@@ -0,0 +1,79 @@
+package cultfit;
+
+public abstract class FitnessProgram {
+    private int id;
+    private FitnessProgramType type;
+    private int size;
+
+    public int getId() {
+        return id;
+    }
+
+    public FitnessProgramType getType() {
+        return type;
+    }
+
+    public FitnessProgram(int id, FitnessProgramType type) {
+        this.id = id;
+        this.type = type;
+        this.size = Constants.CLASS_SIZE;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    public static FitnessProgram addProgram(int id, FitnessProgramType fitnessProgramType) {
+        switch (fitnessProgramType) {
+            case CARDIO:
```

Comment:
this doesnt seem to be right fit here. it should be abstract.

Reply:
Didn't get you. Can u pls add more to your comment ?  @guptashailesh92